thinkbox-deadline/azuredeploy.parameters.json

```diff
@@ -1,24 +1,33 @@
-{
-  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
-  "contentVersion": "1.0.0.0",
-  "parameters": {
-    "location": {
-      "value": "West US"
-    },
-    "publicDnsName": {
-      "value": ""
-    },
-    "newStorageAccount": {
-      "value": "deadline"
-    },
-    "adminUsername": {
-      "value": ""
-    },
-    "adminPassword": {
-      "value": ""
-    },
-    "numberOfSlaves": {
-      "value": 2
-    }
-  }
-}
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "location": {
+      "value": "West US"
+    },
+    "clusterName": {
+      "value": ""
+    },
+    "publicDnsName": {

```

Comment:
For DNS and storage account, you can fill in the value "#####". This will allow us to generate a random string since these need to globally unique and enable automated continuous testing of templates.